reference-tests/src/test/resources/templates/BlockchainReferenceTest.java.template

```diff
@@ -33,23 +37,24 @@ import static org.junit.jupiter.api.Assumptions.assumeTrue;
 /** The blockchain test operation testing framework entry point. */
 @ExtendWith(ReferenceTestWatcher.class)
 @Tag("BlockchainReferenceTest")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Execution(ExecutionMode.CONCURRENT)
 public class %%TESTS_NAME%% {
 
   public ReferenceTestWatcher testWatcher = new ReferenceTestWatcher();
 
   private static final String FAILED_MODULE = "%%FAILED_MODULE%%";
   private static final String FAILED_CONSTRAINT = "%%FAILED_CONSTRAINT%%";
-
   private static final String[] TEST_CONFIG_FILE_DIR_PATH = new String[] {%%TESTS_FILE%%};
-  private static final String FAILED_TESTS_FILE_PATH = "%%FAILED_TEST_FILE_PATH%%";
 
-  public static Stream<Arguments> getTestParametersForConfig() {
+  public static Stream<Arguments> getTestParametersForConfig()
+      throws ExecutionException, InterruptedException {
```

Comment:
Doew generateTestParametersForConfig throw these exceptions?

Reply:
This was due to calling `get()` on a completable future, I've reworked the usage and this is no longer required.